Fedora-37/Dockerfile

```diff
@@ -0,0 +1,121 @@
+# Dockerfile for building container images for use in the EDK2 CI.
+#
+# Copyright (C) 2022, Red Hat, Inc.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+# This file contains the definitions for images to be used for different
+# jobs in the EDK2 CI pipeline. The set of tools and dependencies is split into
+# multiple images to reduce the overall download size by providing images
+# tailored to the task of the CI job. Currently there are two images: "build"
+# and "test".
+# The images are indented to run on x86_64.
+
+
+# Build Image
+# This image is intended for jobs that compile the source code and as a general
+# purpose image. It contains the toolchains for all supported architectures, and
+# all build dependencies.
+FROM registry.fedoraproject.org/fedora-minimal:37 AS build
+ARG GCC_VERSION=12.2.1-4.fc37
+ARG GCC_VERSION_CROSS=12.2.1-2.fc37
+ARG NASM_VERSION=2.15.05-3.fc37
+ARG PYTHON_VERSION=3.11
+ARG GCC_LOONGARCH64_URL="https://github.com/loongson/build-tools/releases/download/2022.09.06/loongarch64-clfs-6.3-cross-tools-c-only.tar.xz"
+ARG CSPELL_VERSION=5.20.0
+ARG MARKDOWNLINT_VERSION=0.31.0
+RUN microdnf \
+      --assumeyes \
+      --nodocs \
+      --setopt=install_weak_deps=0 \
+      install \
+        acpica-tools \
+        gcc-c++-${GCC_VERSION} \
+        gcc-${GCC_VERSION} \
+        gcc-aarch64-linux-gnu-${GCC_VERSION_CROSS} \
+        gcc-arm-linux-gnu-${GCC_VERSION_CROSS} \
+        gcc-riscv64-linux-gnu-${GCC_VERSION_CROSS} \
+        git \
+        libX11-devel \
+        libXext-devel \
+        libuuid-devel \
+        make \
+        nuget \
```

Comment:
I don't think it's a problem, but any reason 37 doesn't have PowerShell like 35? I believe it is used in some azure pipelines.